openapi.yml

```diff
@@ -0,0 +1,105 @@
+openapi: 3.0.3
+info:
+  version: 0.0.1
+  title: Order Book API
+servers:
+- url: http://localhost:8080
+  description: Local
+paths:
+  /api/v1/order:
+    post:
+      summary: Create a new order.
+      responses:
+        201:
+          description: Order has been accepted.
+      requestBody:
+        description: The order to create.
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/Order"
+  /api/v1/orders:
+    get:
+      summary: Get existing orders.
+      # TODO: Later we could add filters through the url query to only get orders matching specific tokens etc.
+      responses:
+        200:
+          description: existing orders
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/Order"
```

Comment:
Should we leave this out for now, as we don't know where it will be stored?

Reply:
Do you mean how we will persist it on disk? I don't think that's relevant for the api because the orderbook service will be an abstraction over the underlying storage. So the solver only needs to know how to communicate with the orderbook hopefully.